src/modules/users/users.service.ts

```diff
@@ -157,6 +165,78 @@ export default class UserService {
     });
   }
 
+  public async generateNumericUniqueId(): Promise<string> {
+    let clubId = '';
+    let existingClub: Clubs | null;
+    do {
+      // Generate a random 10-digit number
+      const randomNumeric = Math.floor(Math.random() * 9000000000) + 1000000000;
+      clubId = `${randomNumeric}`;
+
+      // Check if the generated ID already exists in the database
+      existingClub = await prisma.clubs.findFirst({
+        where: {
+          clubId: clubId,
```

Comment:
pass id

Reply:
I've changed the get club for the meantime, but haven't tested it yet since there's no data on the club. Probably going to change it in the future when the needs change.

https://github.com/Zeff01/formapp-be/blob/66219e291828df2daa71f8f996959c0b61301e12/src/modules/users/users.service.ts#L190-L204